controllers/datadogagent/utils.go

```diff
@@ -1761,6 +1745,14 @@ func getVolumeMountsForSecurityAgent(dda *datadoghqv1alpha1.DatadogAgent) []core
 		}...)
 	}
 
+	if runtimeEnabled {
```

Comment:
/etc/datadog-agent/runtime-security.d is not used by security-agent, only by system-probe

Reply:
I have added here, if later on it is the case.